.circleci/config.yml

```diff
@@ -16,7 +29,32 @@ jobs:
       - run:
           name: Verify
           command: make ci
+      - run:
+          name: Build collector 
+          command: make otelcontribcol
```

Comment:
`make ci` includes `make otelcontribcol`. Why do we need this here?

Reply:
I didn't realize it did. I can remove this but probably better to split it out and keep checks/tests and build separate.

Reply:
Don't have a strong preference either way.

Reply:
I agree, splitting would be better. Something like `ci-build` and `ci-test`. Can be done in a separate PR.